rest/src/main/java/io/syndesis/rest/v1/operations/Creator.java

```diff
@@ -30,7 +32,7 @@
     @POST
     @Produces(MediaType.APPLICATION_JSON)
     @Consumes("application/json")
-    default T create(@NotNull @Valid T obj) {
+    default T create(@Context SecurityContext sec, @NotNull @Valid T obj) {
```

Comment:
Is this needed here?

Reply:
Oh scratch that, it's used in subclass. Hmm not exactly liking this, but can't see a way round it I guess.